bookmyroom/views.py

```diff
@@ -89,46 +93,51 @@ def logout(request):
 def book_new(request):
     if request.method == "POST":
         form = BookForm(request.POST)
-        if form.is_valid():
-            # old_booking=Room_Booking.objects.get()
-            room_booking = form.save(commit=False)
-            room_booking.user = User.objects.get(id=request.COOKIES['user_id'])
-            room_booking.book_time = timezone.now()
-            room_booking.room_status = True  # Implement booking clashes you jackass
-            data = form.cleaned_data
-            # if validate_booking(data, room_booking) == 1:
-            #     pass
-            # else:
-            #     return render(request, 'bookmyroom/book_edit.html', {'error': 1,'username':room_booking.user.name,'in_time':room_booking.in_time,'out_time':room_booking.out_time,'form':form})
-            room_booking.save()
-            return redirect('my_bookings')
+        if form.is_valid() and form.validate_form():
+            #old_booking=Room_Booking.objects.get()
+            # room_booking = form.save(commit=False)
+            # room_booking.user = User.objects.get(id=request.COOKIES['user_id'])
+            # room_booking.book_time = timezone.now()
+            # room_booking.room_status = True  # Implement booking clashes you jackass
+            #data = form.cleaned_data
+            # if validate_booking(data, Room_Booking) == 1:
+            pass
+        else:
+             return render(request, 'bookmyroom/book_edit.html', {'error': 1,'username':room_booking.user.name,'in_time':room_booking.in_time,'out_time':room_booking.out_time,'form':form})
```

Comment:
Are you sure this works, because since there is no room_booking variable so how are we returning it to render